src/pages/Category.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Category = () => (
-  <div className="flex justify-center align-middle">
-    <button className="bg-blue-500 cursor-pointer mr-20 mt-14 mb-14 text-sm px-2 py-1 text-white font-bold" type="button">CHECK STATUS</button>
-  </div>
-);
+const Category = () => {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.category);
+
+  return (
+    <div className="flex justify-center align-middle">
+      <button onClick={() => dispatch(checkStatus())} className="bg-blue-500 cursor-pointer mr-20 mt-14 mb-14 text-sm px-2 py-1 text-white font-bold" type="button">CHECK STATUS</button>
+      <h1>{status}</h1>
+    </div>
```

Comment:
- In your "src/redux/categories/categories.js", I will like to kindly suggest that when the user clicks on the button `Check status`, it returns the phrase `Under Construction`. 🌸